.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.6.5
```

Comment:
v 0.6.9 is available since last week. Do you want to keep the 0.6.5 version?

Reply:
I am updating it to the latest `v0.7.3`.